src/main/java/com/google/api/codegen/configgen/ConfigGeneratorApi.java

```diff
@@ -51,13 +48,14 @@ protected void process() throws Exception {
     }
 
     String outputPath = options.get(OUTPUT_FILE);
-    Map<String, Doc> outputFiles = generateConfig(outputPath);
+    Map<String, String> outputFiles = generateConfig(outputPath);
     ToolUtil.writeFiles(outputFiles, "");
   }
 
-  private Map<String, Doc> generateConfig(String outputPath) {
-    ViewModel viewModel = new ConfigTransformer().generateConfig(model, outputPath);
-    return ImmutableMap.of(
-        outputPath, new CommonSnippetSetRunner(new CommonRenderingUtil()).generate(viewModel));
+  private Map<String, String> generateConfig(String outputPath) {
```

Comment:
Since we're switching out the implementation in this PR, can we remove the transformer classes at the same time? It will be easier to review what code is new, and what has been moved.

Reply:
They are still being used by disco